### DISCUSSION TITLE
Keycloak Custom Attributes not shown in version 24.0.1

### DISCUSSION THREAD

**Discussion by syedshahwar7** [keycloak/keycloak#28316] [Archive]
Custom attributes tab in user is not there in Keycloak version 24.0.1

**Comment by codespearhead** [comment]
@syedshahwar7 You have it enable `Unmanaged Attributes` in the `Realm Settings` for the tab to appear.

**Reply by syedshahwar7** [reply]
Thank you, it worked.

**Reply by codespearhead** [reply]
You're welcome! Can you [mark it as the answer to the discussion](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer)?
